src/components/Personalization/index.js

```diff
@@ -128,15 +103,9 @@ const createPersonalization = ({ config, logger, cookieJar }) => {
 
         const fragments = response.getPayloadsByType(DECISIONS_HANDLE);
 
-        // On response we first hide all the elements for
-        // personalization:page handle
-        hideElementsForPage(fragments);
```

Comment:
Sorry if I'm being dumb, but what functionality did we lose by removing this? I'm not sure I follow the hiding/showing process.

Reply:
Removed

Reply:
Initially the `prehidingSelector` was part of the `event` now since I moved it to `action` this is no longer required. Each `action` handles this internally.

So thanks for pointing that out, now it's much, much cleaner.